### PR TITLE
Remove empty statements

### PR DESCRIPTION
**Pull request by wkaluza**

### CODE REVIEW

File: libs/ledger/src/protocols/main_chain_rpc_service.cpp [hunk 1]
```diff
@@ -436,7 +436,8 @@ State MainChainRpcService::OnWaitForBlocks()
 
     state_machine_->Delay(std::chrono::milliseconds{100 * consecutive_failures_});
     return State::REQUEST_NEXT_BLOCKS;
-  case PromiseState::SUCCESS:;
+  case PromiseState::SUCCESS:
+    break;
```
Comment (denis-trapeznikov) [inline libs/ledger/src/protocols/main_chain_rpc_service.cpp:440]: What exactly does this `break` break?

**Review by denis-trapeznikov — DISMISSED** [review]

File: libs/ledger/src/protocols/main_chain_rpc_service.cpp (hunk 1 above)
Comment (wkaluza) [reply libs/ledger/src/protocols/main_chain_rpc_service.cpp:440]: Prevents accidental fallthrough if someone extends this `switch` in the future. It's less readable to just have a semicolon.

Comment (denis-trapeznikov) [reply libs/ledger/src/protocols/main_chain_rpc_service.cpp:440]: > It's less readable to just have a semicolon.

Questionnable.

Comment (ejfitzgerald) [reply libs/ledger/src/protocols/main_chain_rpc_service.cpp:440]: I fully agree with @wkaluza - this change is much more readable I think that we should add this as an entry in our style guide.

**Review by ejfitzgerald — APPROVED** [review]

**Pull request merged**